Resolve remaining Vale warnings (#57141)
gravitational/teleport: docs/pages/enroll-resources/mcp-access/troubleshooting.mdx

      - "{{internal.mcp_tools}}"
```

> You can configure this user trait with `tctl`, assigning <Var name="my_user" />
> to the name of the Teleport user you want to update:

Reviewer: ```suggestion
You can configure this user trait with `tctl`, assigning it to the Teleport user <Var name="my_user" />:
```
